components/turret.py

```diff
@@ -32,7 +32,8 @@ class Turret:
     # max rotation either side of zero
     MAX_ROTATION = math.radians(200)
 
-    allowable_error = magicbot.tunable(math.radians(20))  # radians
+    allowable_position_error = magicbot.tunable(math.radians(10))  # radians
+    allowable_velocity_error = magicbot.tunable(100)  # counts/100ms
```

Comment:
Definitely a good idea, but would the units here be confusing?

Reply:
Changed to turret rev/s